shinydashboard/lantern/ui.R

```diff
@@ -342,6 +402,28 @@ ui <- dashboardPage(
     ),
     uiOutput("htmlFooter"),
     tags$script(HTML("
+
+      // Set the role of Lantern's main content containing element to 'main'
+      let elems = document.getElementsByClassName('content-wrapper');
+      elems[0].setAttribute('role', 'main');
+      elems[0].id = 'content'
+
+
+      // Create a new li element with all the same attributes as existing side menu element
+      var e = document.getElementById('side_menu');
+      var d = document.createElement('li');
+      d.classList.add('sidebarMenuSelectedTabItem', 'shiny-bound-input');
+      d.dataset.value = e.dataset.value;
+
+      // Replace old div side menu element with new li element created above
+      e.parentNode.replaceChild(d, e);
+      e.remove();
+      d.id = 'side_menu';
+
+      /*
+        // Create an observer that watches if an element's tabindex attribute has been set or altered
+        // If the altered attribute is the tabindex attribute, and its value is not -5, remove the tabindex attribute
+      */
```

Comment:
It might be helpful to elaborate a little more to explain why the tabindex is removed, so that future people looking at this don't think we're doing it just for fun lol